simpleui-server/src/simpleui-server.ts

```diff
@@ -83,7 +108,10 @@ export class SimpleUIServer {
             appName: '',
             webPort: '2080',
             xmlInFile: '',
-            jsonOutFile: ''
+            jsonOutFile: '',
+            zmqHostname: '',
+            DBname: '',
```

Comment:
should be `dbName` for consistency

Reply:
Also could argue that 'theme' should be `themeName` for similar reasons

Reply:
huh thats odd. I definitely pushed it and I can see it [here](https://github.com/greg-ses/simpleui/blob/make-cli-override-option/simpleui-server/src/simpleui-server.ts#L102). Maybe github is behind a little

Reply:
I'm talking about changing `DBname`->`dbName`, not `themeName`

Reply:
oops linked the wrong line. The line above `themeName` is correct and I cant find any instances of `DBname` in the repo. Is it still there for you?

Reply:
Yeah, for example: https://github.com/greg-ses/simpleui/blame/make-cli-override-option/simpleui-server/src/interfaces.ts